frontend/src/Components/CommunityProjects/Projects.css

```diff
@@ -1,44 +1,44 @@
 .project {
-	width: 99%;
-	box-shadow: 0 .2vw .3vw 0 rgba(0, 0, 0, .3), 0 .2vw .3vw 0 rgba(0, 0, 0, .3);
-	transition: box-shadow .2s ease-in-out;
-	padding: 1vw;
-	margin-bottom: 2vw;
-  transition: opacity .2s ease-in-out;
+  display: flex;
```

Comment:
I added only flex setting in one class. Rest of the changes looks like are being done by my prettier extension. No other changes are made to this file.